### DISCUSSION TITLE
Download speed is too slow while downloading lineage

### DISCUSSION THREAD

**Discussion by ghost** [waydroid/waydroid#574] [Q&A]
after running `waydroid init`, it downloads this file.
```
https://sourceforge.net/projects/waydroid/files/images/system/lineage/waydroid_x86_64/lineage-18.1-20221030-VANILLA-waydroid_x86_64-system.zip
```

Downloading speed was too slow despite having my good internet connection, it was giving 100-200 kbps speed.
so  I downloaded it directly from that website. but I am now unable to figure out where I should keep this file and how to proceed further.

Thanks.

**Comment by cniw** [comment]
Cancel it Ctrl+C and then try again run `sudo waydroid init`, do this again and again until you get good download speed like you expect.
